src/VisualStudio/IntegrationTest/TestUtilities/InProcess/InProcComponent.cs

```diff
@@ -29,13 +29,13 @@ private static Dispatcher CurrentApplicationDispatcher
             => Application.Current.Dispatcher;
 
         protected static void BeginInvokeOnUIThread(Action action)
-            => CurrentApplicationDispatcher.BeginInvoke(action);
+            => CurrentApplicationDispatcher.BeginInvoke(action, DispatcherPriority.Background);
```

Comment:
A helper for this seems useful, so that new code will also be Background by default.

Reply:
➡️ This _is_ the helper 😄